### ISSUE TITLE
b- Invoke modules by URI, with optional external variables

### ISSUE THREAD

**Issue by chrisrhayes** [overstory/marklogic-intellij-plugin#4]

**Comment by ronhitchens** [comment]
This depends on #25.  Doing that first.

**Comment by ronhitchens** [comment]
This is pretty much done now.  Modules can be invoked, and IntelliJ will follow through library modules.  This requires specifying where in the project source tree MarkLogic is pointing for its appserver root.  If the ML appserver root is not visible to IntelliJ, it's debugger will get lost when execution stops at a breakpoint.  That may be eventually fixable, but not now.  

Still a little tidy up to do before progressing this ticket.

**Comment by ronhitchens** [comment]
Calling this one complete.  Should work now for any main module in the MarkLogic appserver root path.

**Comment by ronhitchens** [comment]
Addressed in version 0.9.0-beta
